src/Middleware/ConcurrencyLimiter/test/PolicyTests/ResettableBooleanCompletionSourceTests.cs

```diff
@@ -11,10 +11,10 @@ namespace Microsoft.AspNetCore.ConcurrencyLimiter.Tests.PolicyTests
 {
     public static class ResettableBooleanCompletionSourceTests
      {
-        private static StackPolicy _testQueue = TestUtils.CreateStackPolicy(8);
+        private static readonly StackPolicy _testQueue = TestUtils.CreateStackPolicy(8);
 
         [Fact]
-        public async static void CanBeAwaitedMultipleTimes()
+        public static async void CanBeAwaitedMultipleTimes()
```

Comment:
😢

Reply:
Ug, please fix all three of these.
```suggestion
        public static async Task CanBeAwaitedMultipleTimes()
```